.github/workflows/ci.yml

```diff
@@ -12,7 +12,9 @@ jobs:
           2.7,
           3.5,
```

Comment:
Python 3.5 was reached end-of-life last year. 
- https://www.python.org/downloads/release/python-3510/